examples/webgl_raycast_sprite.html

```diff
@@ -0,0 +1,177 @@
+
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>three.js webgl - raycast - sprite</title>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+        <style>
+            body {
+                margin: 0px;
+                background-color: #fff;
+                overflow: hidden;
+            }
+
+            #info{
+                position: absolute;
+                z-index: 1;
+                width: 100%;
+                text-align: center;
+            }
+        </style>
+    </head>
+<body>
+    <div id="info"><a href="http://threejs.org" target="_blank" rel="noopener">three.js</a> - webgl raycast sprite<br></div>
+    <script src="../build/three.js"></script>
+    <script src="./js/controls/OrbitControls.js"></script>
+    <script>
+        var renderer;
+        var scene;
+        var spriteGroup;
+        var camera;
+        var controls;
+
+        init();
+
+        function init() {
+
+            initRender();
```

Comment:
I suggest you don't make all this functions (`initRender`, `initScene`...) but put the code to directly into `init()`. In this way the example is more consistent to the others.